Applications/Calendar/Calendar.cpp

```diff
@@ -0,0 +1,55 @@
+#include "Calendar.h"
+#include <time.h>
+
+String Calendar::get_name_of_month(int month)
+{
+    const String month_names[] = { "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec" };
+    return month_names[month - 1];
+}
+
+int Calendar::get_day()
+{
+    time_t now = time(nullptr);
+    auto* tm = localtime(&now);
+    return tm->tm_mday;
+}
+
+int Calendar::get_month()
+{
+    time_t now = time(nullptr);
+    auto* tm = localtime(&now);
+    return tm->tm_mon + 1;
+}
+
+int Calendar::get_year()
+{
+    time_t now = time(nullptr);
+    auto* tm = localtime(&now);
+    return tm->tm_year + 1900;
+}
+
+bool Calendar::is_today(int year, int month, int day)
+{
+    return day == get_day() && month == get_month() && year == get_year();
```

Comment:
This will break if we roll over midnight between the different function calls.
We should only query the current time once. And for that, please use `Core::DateTime::now()`.